src/frontends/pytorch/src/transforms/aten_stack_list_construct_replacer.cpp

```diff
@@ -38,22 +39,28 @@ AtenStackListConstructReplacer::AtenStackListConstructReplacer() {
         auto axis_node = pattern_map.at(axis).get_node_shared_ptr();
         auto axis_const = std::dynamic_pointer_cast<v0::Constant>(axis_node);
         auto axis = axis_const->cast_vector<int64_t>();
+        OPENVINO_ASSERT(axis.size() == 1);
```

Comment:
nit: move check to pattern

Reply:
I don't want to suppress the error message in this case.

Reply:
I don't like to throw exception in transformation pipeline. Yes we have something wrong here, but better just return false in that case and add message to fw_node using `add_exception_to_fw_node` helper